docs/tutorial-smart-ai-1.md

```diff
@@ -0,0 +1,208 @@
+# Introduction to SmartAI
```

Comment:
This first header should be the same as the file name as it shows in the web tab. So I would change the file name to /introduction-to-smartai.md and keep this header

Reply:
at the very top of the file you can add

```
---
tableofcontents: 1
---
```
to add an automated ToC to the file